src/schemata/definitions/resourceTag.yml

```diff
@@ -0,0 +1,32 @@
+type: string
+title: Resource tag
+description: |
+  This should map to the tag of the resources we have in our API/V2 documentatio
+enum:
```

Comment:
This list intentionally leaves out certain API resources?

Reply:
Resources like `attachments, connector_reset_drivers, group_resources, group_users, port_bindings, schemas, view_resets, views, workflow_run_statuses, workflow_step_substep_completions, workflow_step_substep_repeats, workflow_step_triggers` are omitted because this action does not support them.